### ISSUE TITLE
Homebrew formula using outdated version of stdlib?

### ISSUE THREAD

**Issue by madgen** [agda/agda-stdlib#226]
Homebrew formula [0] uses 477ba28360133b1f5c45ce1b4e6b4efd467af331 which is older than the last release but newer than the penultimate one. Should it be updated? If so to what?

[0] https://github.com/Homebrew/homebrew-core/blob/master/Formula/agda.rb

**Comment by MatthewDaggitt** [comment]
Presumably it should be updated to the commit at tag `0.14` i.e. db16080.

Is it as simple as submitting a pull request with an updated commit hash for the library?

**Comment by madgen** [comment]
Yup, though I'd do an install from the fork before submitting a PR.

**Comment by MatthewDaggitt** [comment]
I've looked into this and unfortunately I don't have access to a Mac so I can't install `brew` myself, and so can't follow [these](https://github.com/Linuxbrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md) instructions on updating formulae.

Could someone else who has a Mac do this? I think all that needs changing is
```
    resource "stdlib" do
      url "https://github.com/agda/agda-stdlib.git",
          :revision => "477ba28360133b1f5c45ce1b4e6b4efd467af331"
```
to the URL of the `v0.15` tag:
```
    resource "stdlib" do
      url "https://github.com/agda/agda-stdlib.git",
          :revision => "c06437e4ebb5365d502fb0a79775e5c591ab8ae5"
```

**Comment by madgen** [comment]
When I created this pull request v0.14 was out and the HEAD of stdlib was incompatible with Agda 2.5.3. Does v0.15 actually work with 2.5.3?

**Comment by MatthewDaggitt** [comment]
Yes, this has been fixed (see the discussion in #227). Version 0.15 is fully compatible with Agda 2.5.3.

**Comment by MatthewDaggitt** [comment]
Thanks @madgen !
